dataframe-jdbc/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/readJdbc.kt

```diff
@@ -523,16 +520,14 @@ private fun buildSchemaByTableColumns(tableColumns: MutableList<TableColumnMetad
     }
 
     return DataFrameSchemaImpl(
-        columns = schemaColumns
+        columns = schemaColumns,
     )
 }
 
-private fun generateColumnSchemaValue(
-    dbType: DbType,
-    tableColumnMetadata: TableColumnMetadata
-): ColumnSchema = dbType.convertSqlTypeToColumnSchemaValue(tableColumnMetadata) ?: ColumnSchema.Value(
-    makeCommonSqlToKTypeMapping(tableColumnMetadata)
-)
+private fun generateColumnSchemaValue(dbType: DbType, tableColumnMetadata: TableColumnMetadata): ColumnSchema =
+    dbType.convertSqlTypeToColumnSchemaValue(tableColumnMetadata) ?: ColumnSchema.Value(
+        makeCommonSqlToKTypeMapping(tableColumnMetadata),
```

Comment:
sorry, but it's fine with signatures, but not fine with the client code for passing parameters, it breaks the reading for me

Reply:
Yes, this can indeed look better. How about:
```kt
dbType.convertSqlTypeToColumnSchemaValue(tableColumnMetadata) 
    ?: ColumnSchema.Value(makeCommonSqlToKTypeMapping(tableColumnMetadata))
```
or
```kt
dbType.convertSqlTypeToColumnSchemaValue(tableColumnMetadata) 
    ?: ColumnSchema.Value(
          makeCommonSqlToKTypeMapping(tableColumnMetadata),
    )
```
?